packages/common/src/store/ui/mobile-overflow-menu/types.ts

```diff
@@ -36,6 +37,7 @@ export enum OverflowSource {
 export type OpenOverflowMenuPayload = {
   source: OverflowSource
   id: ID | string
+  contextPlaylistId?: ID
```

Comment:
would it make sense to just have a "context" field that any overflow action can add to? feels a little awkward that we are adding "playlist" word into a somewhat generic function?

Reply:
True yea I was thinking about too, but ultimately feel like this is a bit safer so that we can check for a specific key instead of having to check everywhere this is used what the value is and if its applicable